src/models/SDORdfConcept.py

```diff
@@ -406,7 +408,7 @@ def __create_skos_concept(
 
         return skos_concept_node
 
-    def __payload_to_rdf(self, payload, parent_node, class_uri):
+    def __payload_to_rdf(self, payload, parent_node, class_uri):  # noqa: C901 #TODO
```

Comment:
advice is to refactor this function/method a bit to make it less complex or error prone.

Reply:
Great that you have configured it this way. It doesn't break things, but reminds us of necessary (but not urgent) improvements.